### PR TITLE
Test tox job

### PR DESCRIPTION
**Pull request by sloria**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[tox]
+envlist =
```
Comment (asottile) [inline tox.ini]: skipsdist=true here

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  py27
+  py34
+  py35
+  py36
+  py37
+  py38
+  pypy2
+  pypy3
+  docs
```
Comment (asottile) [inline tox.ini]: I prefer these on one line

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+[testenv]
+skip_install = true
```
Comment (asottile) [inline tox.ini]: can delete this after adding `skipsdist`

File: tox.ini [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+commands = echo "success 🎉"
```
Comment (asottile) [inline tox.ini]: `python --version --version` would be more useful

**Comment by sloria** [comment]
Done.

File: azure-pipelines.yml [hunk 5]
```diff
@@ -6,3 +6,15 @@ trigger:
...
+    toxenvs:
+      - py27
+      - py34
+      - py35
+      - py36
+      - py37
+      - py38
+      - pypy2
+      - pypy3
+      - docs
```
Comment (asottile) [inline azure-pipelines.yml]: this'll need `coverage: false` -- I also prefer the inline list here for `toxenvs: [py27, py34, ...]`

File: tox.ini [hunk 6]
```diff
@@ -0,0 +1,6 @@
+[tox]
+skipsdist = true
+envlist = py27,py34,py35,py36,py37,py38,pypy2,pypy3,docs 
+
+[testenv]
+commands = python --version
```
Comment (asottile) [inline tox.ini]: I did actually mean `--version` twice -- that way we can see the more specific version information

**Comment by sloria** [comment]
Anything else?

**Comment by asottile** [comment]
oops, looks like two last little things :S -- some whitespace and `s/pypy2/pypy/g` should fix everything -- thanks again for adding this!

**Comment by sloria** [comment]
✅ 💸 🚢 🇮🇹

**Pull request merged**